app/src/main/java/tech/ula/utils/ServerUtility.kt

```diff
@@ -47,37 +51,53 @@ class ServerUtility(private val applicationFilesDirPath: String, private val exe
     }
 
     private fun startSSHServer(session: Session): Long {
+        var processPid = -1L
         val targetDirectoryName = session.filesystemId.toString()
         deletePidFile(session)
         val command = "../support/execInProot.sh /bin/bash -c /support/startSSHServer.sh"
-        val process = execUtility.wrapWithBusyboxAndExecute(targetDirectoryName, command, doWait = false)
-        return process.pid()
+        try {
```

Comment:
these could be 
```
return try { ... } 
catch { 
    logger.runtimeErrorForCommand() 
    -1
}
```

Reply:
That's pretty convenient.  I'll change it to use `return try`

Reply:
Gets rid of the `var` handily enough as well.